### PR TITLE
Add unverified notificationType for who boosted the project

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
https://github.com/Giveth/GIVeconomy/issues/749

### CODE REVIEW

**Comment by mohammadranjbarz** [comment]
Obviously after merging this we would drop notification-center DB and create **givethio** entity in third party table again :))

File: migrations/1660716115917-seedNotificationType.ts [hunk 1]
```diff
@@ -1174,6 +1174,34 @@ export const GivethNotificationTypes = {
...
     content: 'You project {project name} is not verified anymore',
   },
+  PROJECT_UNVERIFIED_WHO_BOOSTED: {
+    name: 'Project unverified - Users Who Boosted',
+    microService: MICRO_SERVICES.givethio,
+    category: NOTIFICATION_CATEGORY.PROJECT_RELATED,
+    icon: '',
+    schemaValidator: SCHEMA_VALIDATORS_NAMES.PROJECT_UNVERIFIED_WHO_BOOSTED,
+    emailNotifierService: THIRD_PARTY_EMAIL_SERVICES.SEGMENT,
+    emailNotificationId: null,
```
Comment (CarlosQ96) [inline migrations/1660716115917-seedNotificationType.ts:1184]: Add an email notification ID , and later Mitch or someone else will connect it to autopilot

**Review by CarlosQ96 — APPROVED** [review]
Thanks Mohammad! all looks good but first add the notification ID

File: migrations/1660716115917-seedNotificationType.ts (hunk 1 above)
Comment (CarlosQ96) [reply migrations/1660716115917-seedNotificationType.ts:1184]: Only if this is sending emails with segment.

Comment (mohammadranjbarz) [reply migrations/1660716115917-seedNotificationType.ts]: Yeah you are right, I added it.

**Comment by mohammadranjbarz** [comment]
Thanks @CarlosQ96  I will merge it now

**Pull request merged**